### PR TITLE
Re-organise API route handlers

### PR DESCRIPTION
**Pull request by i-like-robots**
While plugging away at a testing setup for this app I realised we were not able to easily access our route handlers.

This just copies and pastes the `/api/V1/` routes into separate chunks. Adding this PR now to prevent the tests branch growing and growing and becoming unmanageable.

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const cookieParser = require('cookie-parser');
 const path = require('path');
 const authS3O = require('s3o-middleware');
 
-const apiV1 = require('./controllers/api/v1');
+const apiV1 = require('./controllers/api/V1');
```
Comment (ironsidevsquincy) [inline server/app.js]: why capitalise?

Comment (i-like-robots) [inline server/app.js]: Just to match the URL

Comment (ironsidevsquincy) [inline server/app.js]: ah. maybe we should lowercase the url. capitals in filenames make me feel funny

Comment (i-like-robots) [inline server/app.js]: Fortunately the URL is not case sensitive =]

**Pull request merged**